Source/JavaScriptCore/inspector/agents/InspectorDebuggerAgent.cpp

```diff
@@ -471,7 +471,12 @@ void InspectorDebuggerAgent::getScriptSource(ErrorString& error, const String& s
     
 void InspectorDebuggerAgent::setScriptSource(ErrorString& error, const String& scriptID, const String& scriptSource)
 {
-    
+    JSC::SourceID sourceID = scriptID.toIntPtr();
```

Comment:
Hey @smartether,
Webkit has one very handy macro called [UNUSED_PARAM](https://github.com/NativeScript/webkit/blob/ios/Source/WTF/wtf/Compiler.h#L299) so you may rewrite the method in the following way

```
void InspectorDebuggerAgent::setScriptSource(ErrorString& error, const String& scriptID, const String& scriptSource)
{
    UNUSED_PARAM(error);
    UNUSED_PARAM(scriptID);
    UNUSED_PARAM(scriptSource);
}
```
What do you think ? :)